module/zfs/zfs_acl.c

```diff
@@ -2459,7 +2459,7 @@ zfs_fastaccesschk_execute(znode_t *zdp, cred_t *cr)
 	mutex_exit(&zdp->z_acl_lock);
 
 slow:
-	DTRACE_PROBE(zfs__fastpath__execute__access__miss);
+//	DTRACE_PROBE(zfs__fastpath__execute__access__miss);
```

Comment:
It shouldn't be necessary to comment this out anymore, correct?

Reply:
Correct.